src/utils/tokens.js

```diff
@@ -0,0 +1,31 @@
+
+import sendJson from '../tmp_fetch_send_json'
+import { ACCOUNT_HELPER_URL } from './wallet'
+import * as Sentry from '@sentry/browser'
+import { wallet } from './wallet'
+
+export const getLikelyContracts = async (accountId) => (
+    await logAndIgnoreError(sendJson('GET', `${ACCOUNT_HELPER_URL}/account/${accountId}/likelyTokens`))
```

Comment:
:nit: superfluous `await`; no need to have this be an expression at all and the `await` doesn't do anything in this context.